<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="BuildTreeButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>370</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>BuildTreeButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DepthButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>150</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Depth</string>
    </property>
   </widget>
   <widget class="QListWidget" name="Depth">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>180</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>240</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="AddButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SearchButton">
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RemoveButton">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>241</width>
      <height>24</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLineEdit" name="KeyEdit"/>
    <widget class="QLineEdit" name="NameEdit"/>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>220</y>
      <width>241</width>
      <height>75</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QPushButton" name="PreButton">
     <property name="text">
      <string>PreButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PostButton">
     <property name="text">
      <string>PostButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SymButton">
     <property name="text">
      <string>SymButton</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
